ilastik/shell/gui/reportIssueDialog.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+import ilastik
+import platform
+import sys
+import webbrowser
+from PyQt5.QtWidgets import QApplication, QDialog, QVBoxLayout, QLabel, QTextEdit, QPushButton, QHBoxLayout, QSizePolicy
+from urllib.parse import quote
+
+
+def _get_log_since_last_startup() -> str:
+    """
+    Reads the log up to maximum 1 MB from the end and returns the text since the last startup, or the entire 1 MB.
+    We don't want people emailing us more than 1MB of log...
+    """
+    log_path = ilastik.ilastik_logging.default_config.get_logfile_path()
+    marker = b"Starting ilastik"
+    text_size_limit = 1048576  # 1 MB
+    file_size = int(os.path.getsize(log_path))
+    with open(log_path, "rb") as log_file:
+        position = max(0, file_size - text_size_limit)
+        log_file.seek(position)
+        log_text = log_file.read()
+    if marker in log_text:
+        return b"".join(log_text.rpartition(marker)[1:]).decode()
+    else:
+        return log_text.decode()
+
+
+class ReportIssueDialog(QDialog):
+    def __init__(self, parent=None, workflow_report_text=""):
+        super().__init__(parent=parent)
+
+        self.setWindowTitle("Report issue")
+        self.setFixedSize(600, 300)
+
+        main_layout = QVBoxLayout()
+
+        # Create and add the instruction label
+        instruction_label = QLabel()
+        instruction_label.setText(
+            "<p>Sorry you ran into a problem! Please copy the information below, and include it when you "
+            "report your issue.</p>"
+            '<p>The best place to get help is the community forum, using the "ilastik" tag '
```

Comment:
Maybe encourage user to post their issue to `image.sc` first? With motivational wording like "this way you will help other people and make everyone happy".

Reply:
New wording is up, screenshot below or see diff :)